docs/inference.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 🤗 Hub Inference API
+---
+
+<h1>🤗 Hub Inference API</h1>
+
+For detailed usage documentation, please refer to [Accelerated Inference API Documentation](https://api-inference.huggingface.co/docs/python/html/index.html).
+
+## Do you only support 🤗 Transformers?
+
+No, the 🤗 Hub supports other libraries, and we're working on expanding this support!
+
+Find all our supported libraries [here](https://github.com/huggingface/huggingface_hub/blob/main/interfaces/Libraries.ts)!
```

Comment:
this table is awesome but I would put it in its own file `libraries.md`, as it's more general than the inference API, no?

Reply:
Good point. For now I added to /docs/libraries.md, but I would actually suggest to give more visibility in longer term (longer term=few weeks from now). 

I would have a Libraries page just as we have Metrics and Languages. WDYT?